File: week08/18/prob1.c
# include <stdio.h>
# include <stdlib.h>

int solution(int K, int A[], int N) {
    // Implement your solution here

    int count=0;
    int sum=0;

    for (int i=0; i<N; i++){
        sum += A[i];
        if (sum >= K){
            count++;
            sum=0;
        } 
    }

    return count;
}

File: week08/26/prob1.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(K, A):
    # Implement your solution here
    cnt = 1
    # store sum of ropes
    ropes = 0
    for x in A:
        ropes += x
        if ropes >= K:
            cnt += 1
            ropes = 0
    if ropes < K:
        cnt -= 1
    return cnt

